index.js

```diff
@@ -55,6 +55,11 @@ function dispatcher (hooks) {
     if (hooks.wrapInitialState) initialStateWraps.push(hooks.wrapInitialState)
     if (hooks.wrapReducers) reducerWraps.push(hooks.wrapReducers)
     if (hooks.wrapEffects) effectWraps.push(hooks.wrapEffects)
+    if (hooks.injectModels) {
+      hooks.injectModels.forEach(function (model) {
```

Comment:
I think you can just pass `setModel` in here without needing to pass the closure - shaves a stack in the stack trace :D